src/components/accordion/index.md.njk

```diff
@@ -98,11 +98,13 @@ If you’ve decided that you need the summary line, you must make it as short as
 
 The accordion component shows section headings as `<h2>` headings. If needed, change the heading level of the section headings to make them fit within the other headings on the page.
 
-### Change how an accordion starts
+### Configure how an accordion looks when the user first sees it
 
 Users might need some sections to be open from the start. If they leave and then return to the page, they might also need sections they opened to stay open.
 
-If the user leaves and then returns to the page, the accordion component will remember if sections were opened or closed. You can configure sections to start and stay open, but not stay closed.
+If the user leaves and then returns to the page, the accordion component will use ['session storage'](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) to remember if sections were opened or closed. You can configure sections to start and stay open, but not stay closed.
```

Comment:
Do we need to specify that returning-to-the-page = by going back in the browser?

Reply:
What about this?

```suggestion
If the user leaves and then returns to the page within the same page session, the accordion component will use ['session storage'](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) to remember which sections were open. You can configure sections to start and stay open, but not stay closed.
```

Reply:
I do think it's a bit strange that we don't tell users _how_ to configure sections to start and stay open anywhere in the guidance – but maybe that's something we address in a future PR.

Reply:
Yeah, I think it's probably a slight over-simplification, but accurate enough for this purpose.

Reply:
Thanks both. Sounds like we're all happy with the below then?

```suggestion
If the user leaves and then returns to the page within the same tab, the accordion component will use ['session storage'](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) to remember which sections were open. You can configure sections to start and stay open, but not stay closed.
```